edx_salesforce/management/commands/run_user_account_report.py

```diff
@@ -99,7 +99,7 @@ def handle(self, *args, **options):
         )
 
         with open(output_filename, 'wb') as csvfile:  # pylint: disable=open-builtin
-            writer = csv.writer(csvfile, delimiter=',')
+            writer = csv.writer(csvfile, delimiter=str(','))
```

Comment:
I was getting an error on that !!!
TypeError: "delimiter" must be string, not unicode